src/main/java/uk/gov/pay/api/validation/ExactLengthValidator.java

```diff
@@ -1,10 +1,19 @@
 package uk.gov.pay.api.validation;
 
 
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
 import static org.apache.commons.lang3.StringUtils.isBlank;
 
-class ExactLengthValidator {
+class ExactLengthValidator implements ConstraintValidator<ValidCountry, String>  {
```

Comment:
This was a little confusing at first but I see you've re-used an existing `ExactLengthValidator` class and added the implements `ConstraintValidator...`. To make this reusable perhaps create an annotation `@ExactLength` which is validated by this class and then use it within the `@ValidCountry` or you could just use `@Size` in a similar manner and use `@ReportAsSingleViolation`.
```
@Target({ FIELD, METHOD, PARAMETER, ANNOTATION_TYPE, TYPE_USE })
@Retention(RUNTIME)
@Constraint(validatedBy = {})
@Documented
@Size(max = 2, min = 2)
@ReportAsSingleViolation
public @interface ValidCountry {

    String message() default "Must be exactly 2 characters length";

    Class<?>[] groups() default {};

    Class<? extends Payload>[] payload() default {};
}
```